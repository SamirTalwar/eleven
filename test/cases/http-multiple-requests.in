for i in {1..3}; do
  http --timeout=5 --check-status --print='hb' :8080 | dos2unix | grep -Ev '^Date: '
	echo
done
