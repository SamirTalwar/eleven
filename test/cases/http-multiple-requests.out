HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Hello, world!

HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Hello, world!

HTTP/1.1 200 OK
Connection: keep-alive
Transfer-Encoding: chunked

Hello, world!

